src/librustdoc/html/static/rustdoc.css

```diff
@@ -1228,6 +1228,39 @@ h4 > .important-traits {
 	margin: 5px 0;
 }
 
+#sidebar-toggle {
+	position: fixed;
```

Comment:
What was actually changed here? Moving the lines up makes it harder to see the actual diff.

Reply:
Just look at the first commit. I splitted the changed so you can review easily.

Reply:
I am looking at the first commit, it's still harder to tell what was added

Reply:
Ah, the only change is adding the stuff to the media query.